install-with-linux.md

```diff
@@ -54,10 +54,16 @@ When downloading a release, select the brunch...tar.gz file from the assets at t
 3. Once both files have been downloaded, the Brunch release and your chosen ChromeOS recovery, open a terminal. In most Linux distros you can just press **ctrl + alt + t** to open it quickly.
 4. Make sure that pv, cgpt, tar and unzip are installed.
 
-```sudo apt update && sudo apt -y install pv cgpt tar unzip```
-  
-  * My example uses `apt`, a package manager for Debian and Ubuntu based distros. If you use Arch, you will need [vboot-utils][vboot-utils] for access to cgpt and a different package manager may be needed to install the rest.
+On ubuntu/debian/linux mint<br/>
```

Comment:
The Distro names should be capitalized, and there is no colon at the end like on the Arch instructions.